coral/src/app/features/topics/hooks/list/useGetTopics.test.tsx

```diff
@@ -187,4 +187,66 @@ describe("useGetTopics", () => {
       expect(envList).toEqual(["DEV", "DEV", "DEV"]);
     });
   });
+
+  describe("handles responses based on the team", () => {
+    it("returns a list of two topics with `TEST_TEAM_02` team", async () => {
+      mockTopicGetRequest({
+        mswInstance: server,
+      });
+
+      const { result } = await renderHook(
+        () =>
+          useGetTopics({
+            currentPage: 1,
+            environment: "ALL",
+            teamName: "TEST_TEAM_02",
+          }),
+        {
+          wrapper,
+        }
+      );
+
+      await waitFor(() => {
+        expect(result.current.isSuccess).toBe(true);
+      });
+
+      const teamList = result.current.data?.entries
```

Comment:
Ah, you mean instead of checking that the results only contain topics  with a `teamName` matching "TEST_TEAM_02", just test the whole result (aka topic list)? We could do that too, sure! It makes the test longer :D but I guess that's ok.  

I thought using that map here makes the test more readable, but now that I look at it again, I don't think so anymore 😓  because I needed a moment myself to understand what I did.